core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -1068,7 +1068,15 @@ static function ($allowedTag)
 
 						if ($objFile !== null)
 						{
-							$arrCache[$strTag] = System::urlEncode($objFile->path);
+							$staticUrl = '';
+
+							// Prepend static URL if insert tag is used as URL attribute
+							if (\in_array('urlattr', $flags, true))
```

Comment:
I think we should always add the static URL/base URL, even if no `urlattr` flag is set.